deluge/tests/__init__.py

```diff
@@ -0,0 +1,8 @@
+# Increase open file descriptor limit to allow tests to run
+# without getting error: what():  epoll: Too many open files
+import resource
+
+try:
+    resource.setrlimit(resource.RLIMIT_NOFILE, (65536, 65536))
+except ValueError as ex:
+    print "Failed to raise file descriptor limit:", ex
```

Comment:
Please use the print function from __future__: from __future__ import print_function